runtime_parameters.go

```diff
@@ -16,9 +16,44 @@ type RuntimeParameter struct {
 	Value     interface{} `json:"value"`
 }
 
+// RuntimeDefinitions represents rabbitmq runtime configuration.
+type RuntimeDefinitions struct {
```

Comment:
They are called just [definitions](https://www.rabbitmq.com/definitions.html), not runtime definitions. A lot of them have nothing to do with runtime parameters and do not change at runtime (e.g. product version).